scripts/gulp/plugins/util/parseType.ts

```diff
@@ -32,12 +32,12 @@ const normalizeType = (propType: string): string => {
 };
 
 const getTypeFromBabelTree = (componentFile: t.File, componentName: string, propName: string) => {
-  let typeAnnotation: t.TSType;
+  let typeAnnotation: t.TSType | undefined;
 
   const propertyVisitor: Babel.Visitor = {
     TSPropertySignature: path => {
       if (path.get('key').isIdentifier({ name: propName })) {
-        const annotationPath: NodePath<t.TSTypeAnnotation> = path.get('typeAnnotation');
+        const annotationPath = path.get('typeAnnotation') as NodePath<t.TSTypeAnnotation>;
```

Comment:
Without the cast this is `NodePath<t.TSTypeAnnotation | null>` and I wasn't sure how to properly detect/handle the null case.